includes/fields/class-fieldtypes-radiobutton.php

```diff
@@ -72,14 +72,27 @@ public function render_field_inner( &$field, $value, $context, &$extra = null, $
 				$checked = 'checked="checked"';
 			}
 
-			$html .= sprintf(
-				'<div class="%1$s"><label for="wpbdp-field-%5$d-%4$s"><input id="wpbdp-field-%5$d-%4$s" type="radio" name="%2$s" value="%4$s" %3$s /> %4$s</label></div>',
-				implode( ' ', $css_classes ),
-				'listingfields[' . $field->get_id() . ']',
-				$checked,
-				esc_attr( $label ),
-				$field->get_id()
-			);
+			if ( $field->get_association() === 'meta' ) {
+				$html .= sprintf(
+					'<div class="%1$s"><label for="wpbdp-field-%5$d-%4$s"><input id="wpbdp-field-%5$d-%4$s" type="radio" name="%2$s" value="%4$s" %3$s /> %4$s</label></div>',
```

Comment:
I’ve just tried rewriting this condition, and it seems possible. It could look something like the example below. I haven’t tested it yet, but here’s more context:

```php
$html .= sprintf(
    '<div class="%1$s"><label for="wpbdp-field-%6$d-%5$s"><input id="wpbdp-field-%6$d-%5$s" type="radio" name="%2$s" value="%3$s" %4$s /> %5$s</label></div>',
    implode(' ', $css_classes),
    'listingfields[' . $field->get_id() . ']',
    $field->get_association() === 'meta' ? esc_attr($label) : $option,
    $checked,
    esc_attr($label),
    $field->get_id()
);
```